src/ApiGateways/Web.Bff.Shopping/aggregator/Controllers/OrderController.cs

```diff
@@ -1,44 +1,36 @@
-﻿using Microsoft.AspNetCore.Authorization;
-using Microsoft.AspNetCore.Mvc;
-using Microsoft.eShopOnContainers.Web.Shopping.HttpAggregator.Models;
-using Microsoft.eShopOnContainers.Web.Shopping.HttpAggregator.Services;
-using System.Net;
-using System.Threading.Tasks;
+﻿namespace Microsoft.eShopOnContainers.Web.Shopping.HttpAggregator.Controllers;
 
-namespace Microsoft.eShopOnContainers.Web.Shopping.HttpAggregator.Controllers
+[Route("api/v1/[controller]")]
+[Authorize]
+[ApiController]
+public class OrderController : ControllerBase
 {
-    [Route("api/v1/[controller]")]
-    [Authorize]
-    [ApiController]
-    public class OrderController : ControllerBase
+    private readonly IBasketService _basketService;
+    private readonly IOrderingService _orderingService;
+    public OrderController(IBasketService basketService, IOrderingService orderingService)
```

Comment:
```suggestion

    public OrderController(IBasketService basketService, IOrderingService orderingService)
```

Need a space between `.ctor` and field declarations.